core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -64,7 +64,7 @@ private static String replacedSince(String version, String newProperty) {
       put("alluxio.master.connection.timeout", removedSince(V2_0_0));
       put("alluxio.master.file.async.persist.handler", removedSince(V2_0_0));
       put("alluxio.master.heartbeat.interval",
-          replacedSince(V2_0_0, PropertyKey.MASTER_WORKER_HEARTBEAT_INTERVAL.getName()));
+          replacedSince(V2_0_0, PropertyKey.MASTER_LOST_WORKER_FILE_DETECTION_INTERVAL.getName()));
```

Comment:
Yay, it looks like we already deprecated it!

Reply:
We change the name from `alluxio.master.heartbeat.interval` to `alluxio.master.worker.heartbeat.interval`.
Oh... may be I need to add the `alluxio.master.worker.heartbeat.interval` as the alias of this key or add it to the removedKeys

Reply:
Looks like the RemovedKey is updated in batch in release process. I will not update it for now.